tests/debugging/test_encoding.py

```diff
@@ -250,6 +250,73 @@ def test_batch_json_encoder():
     assert queue.count == 0
 
 
+def test_process_tags_are_not_included_by_default():
+    s = Snapshot(
+        probe=create_snapshot_line_probe(probe_id="batch-test", source_file="foo.py", line=42),
+        frame=inspect.currentframe(),
+        thread=threading.current_thread(),
+    )
+    buffer_size = 30 * (1 << 20)
+    queue = SignalQueue(encoder=LogSignalJsonEncoder(None), buffer_size=buffer_size)
+
+    s.line({})
+
+    queue = SignalQueue(encoder=LogSignalJsonEncoder("test-service"))
+    queue.put(s)
+    data = queue.flush()
+    assert data is not None
+    payload, _ = data
+    decoded = json.loads(payload.decode())
+    assert "process_tags" not in decoded[0]
+
+
+def test_process_tags_are_included():
+    from unittest.mock import patch
+
+    from ddtrace.internal.process_tags import _process_tag_reload
+    from ddtrace.internal.process_tags.constants import ENTRYPOINT_BASEDIR_TAG
+    from ddtrace.internal.process_tags.constants import ENTRYPOINT_NAME_TAG
+    from ddtrace.internal.process_tags.constants import ENTRYPOINT_TYPE_SCRIPT
+    from ddtrace.internal.process_tags.constants import ENTRYPOINT_TYPE_TAG
+    from ddtrace.internal.process_tags.constants import ENTRYPOINT_WORKDIR_TAG
```

Comment:
I think here it's enough to test that the field is set. Testing for the actual values should be done in a unit test for the process tags feature itself to avoid redundancy.